compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/symbols/Symbols.java

```diff
@@ -222,6 +222,21 @@ public static BXMLNSSymbol createXMLNSSymbol(Name name,
         return new BXMLNSSymbol(name, nsURI, pkgID, owner, pos, origin);
     }
 
+    public static BVarSymbol createBVarSymbol(long flags,
```

Comment:
Why do we need a separate method for this?

Reply:
We will have to use directly this method instead of `createBVarSymbolForType` when `type.tsymbol` is not available

Reply:
I mean why aren't we directly using the constructor?

Reply:
We can use the constructor but since we're using a method when there is a tsymbol would it be inconsistent since they are both used for tuple members, to have a method some places and a constructor in others?

Reply:
When there's a tsymbol we have additional stuff to do, access owner, pos, etc., in which case doing it wherever the tsymbol is present is repetitive code. Here we don't have that. We are just wrapping the constructor invocation which is not necessary.

Reply:
I'll remove it and use the constructor instead.